index.js

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env node
+require("dotenv").config();
+const axios = require("axios");
+
+// Store Slack access token in a .env file in the root folder
+const accessToken = process.env.SLACK_ACCESS_TOKEN;
+
+async function getConversationId(conversation) {
+
+  // Trim the pound sign if it was provided in the conversation argument
+  if (conversation.startsWith('#')) {
+    conversation = conversation.slice(1)
+  }
+
+  // Destructuring to pull out the channels result from the response
+  const { data: { channels } } = await axios.get(`https://slack.com/api/conversations.list?token=${accessToken}&pretty=1`)
+
+  // Find the channel with matching conversation name and pull out its ID
+  const conversationId = channels.find(chan => chan.name === conversation).id
+
+  return conversationId
+}
+
+async function getConversationMembers(conversationId) {
+
+  async function queryForMembers(membersCollection = [], nextCursor = '') {
```

Comment:
what is nextCursor?

Reply:
I also don't know I like having queryForMembers scoped within the getConversationMembers function

Reply:
nextCursor is following the language of the response from Slack. The results are paginated, and "next_cursor" is the indication that there is another page of results to poll.

Scoping: Absolutely. This is a hold of from refactoring. There's no need for it now. Though a function in a function and taking advantage of closures in this way isn't uncommon, there's no longer a purpose for it now since I changed how I handled it over iterations. Good catch.